#!/usr/bin/env python
PACKAGE = "rcbigcar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("MaxX", double_t, 0, "Max Linear X Speed (m/s)",   5.0, 0, 10)
# gen.add("MaxY", double_t, 0, "Max Linear Y Speed (m/s)",   0.34, 0, 10)
gen.add("MaxW", double_t, 0, "Max Angular Speed (rad/s)",  20.0, 0, 20)

gen.add("MaxAccX", double_t, 0, "Max Linear X Acceleration (m/s^2)",   5, 0, 10)
# gen.add("MaxAccY", double_t, 0, "Max Linear Y Acceleration (m/s^2)",   0.5, 0, 10)
gen.add("MaxAccW", double_t, 0, "Max Angular Acceleration (rad/s^2)",  20, 0, 10)

exit(gen.generate(PACKAGE, "teleop", "Teleop"))
